lib/package-managers/npm.js

```diff
@@ -216,8 +217,8 @@ module.exports = {
         return viewMany(packageName, ['time', 'versions'], currentVersion, {timeout: options.timeout})
             .then(result => {
                 const versions = doesSatisfyEnginesNode(result.versions, options.enginesNode);
-                return _.keys(result.time).reduce((accum, key) =>
-                    accum.concat(_.includes(TIME_FIELDS, key) || _.includes(versions, key) ? key : []), []
+                return Object.keys(result.time || {}).reduce((accum, key) =>
```

Comment:
I was getting `result.time` undefined in some of the current tests hence the fallback.